webpack.config.js

```diff
@@ -105,7 +105,6 @@ module.exports = {
     ].filter(Boolean),
     devtool: prod ? false : "source-map",
     devServer: {
-        port: 4000,
```

Comment:
By hardcoding this port here, we remove the functionality to configure it with either the PORT env var or with the --port CLI arg.